File: C++/1009/1009.cpp
#include <iostream>
#include <string>
using namespace std;

double salary, sale_total, TOTAL;
string name;

int main() {
	// String user input
	cout << "";
	cin >> name;	
	// Integer user input
	scanf("%lf", &salary);
	scanf("%lf", &sale_total);
	
	double sale_percentage = 0.15 * sale_total;	
	TOTAL = sale_percentage + salary;

	printf("TOTAL = R$ %.2lf\n", TOTAL);
}

File: Pascal/1009/1009.pas
var
	salary, sale_total, total : double;
	name : string;
begin
	readln(name);
	readln(salary);
	readln(sale_total);

	total := (0.15 * sale_total) + salary;

	writeln('TOTAL = R$ ', total:2:2);
end.
